config/service.go

```diff
@@ -55,6 +55,10 @@ type ServiceTelemetryLogs struct {
 	// Encoding sets the logger's encoding.
 	// Valid values are "json" and "console".
 	Encoding string `mapstructure:"encoding"`
+
+	// EnableGrpcLogs replaces grpc middleware default logging with the collectors
+	// logger resulting in more detailed messages for grpc connections
+	EnableGrpcLogs bool `mapstructure:"grpc"`
```

Comment:
We should not have this here, if you need a configuration for this, that should go in the configgrpc not in the "service" logger.

https://github.com/open-telemetry/opentelemetry-collector/pull/4401/files#diff-
93a7a07281367bed9cd2555be4655e6de0ee0750cffbe9789e03294f0f9e6c84R45

Reply:
Also the name of the property does not make too much sense anyway, something like "enable_for_grpc" would have been better.

Reply:
Will do.